addons-tutorial.md

```diff
@@ -0,0 +1,91 @@
+Addons are used to customize the images used for the containers. An addon has *hooks* that are
+invoked at various points in the lifecycle of a container. A good example is an addon which
+enables the SSH service inside a container to provide access from the outside for any kind
+of automation of the container.  
+
+Addons are created independently, and can be attached to individual applications.
```

Comment:
I think this will work nicely as a tutorial, great idea! :)
We don't have any "real" tutorials in the docs yet, so there isn't much to refer to. But overall, we don't want a lot of explanation in here - we just want to guide users through the steps to get a result.

So the introduction should be a bit more direct. Not explaining why addons are nice and what you can do with them, but telling users that in this tutorial, they will create an addon with a hook that enables the SSH service.